kore/test/Test/Kore/BugReport.hs

```diff
@@ -82,6 +90,40 @@ test_parse =
             actual = expect >>= parseKoreLogOpts . unparseKoreLogOptions
         getParseResult expect === getParseResult actual
 
+test_bugReportOption :: TestTree
+test_bugReportOption =
+    testGroup
+        "Parse BugReportOption"
+        [ testCase "Parse BugReportEnable" enableAssert
+        , testCase "Parse BugReportDisable" disableAssert
+        , testCase "Parse BugReportOnError" onErrorAssert
```

Comment:
I suggest adding a function to the `where` clause,

```.hs
test :: TestName -> [String] -> BugReportOption -> TestTree
test testName arguments opt =
    testCase testName $ assertParse arguments opt
```

Then the test cases can be written as:

```suggestion
        , test "Parse BugReportOnError" [] BugReportOnError
```

This way, it's easy to scan the list of test cases and we don't have to add `HasCallStack` to `enableAssert`, `disableAssert`, and `onErrorAssert`.